test-suite/handwritten-src/js/CppExceptionTest.js

```diff
@@ -8,15 +8,26 @@ class CppExceptionTest {
     }
 
     testCppException() {
-        var thrown = null;
         try {
             this.cppInterface.throwAnException();
+            fail();
         } catch (e) {
-            thrown = e;
+            assertNe(thrown, null);
```

Comment:
use undefined var?

Reply:
I'm sure once I can run tests I'll be able to sort these things out. Also not sure if there is a ton of value anyway in maintaining the javascript ones if we have typescript